themes/default/content/docs/get-started/install/_index.md

```diff
@@ -129,6 +129,24 @@ Subsequent updates can be installed in the usual way:
 > choco upgrade pulumi
 ```
 
+### Winget
+
+Install Pulumi using [winget-cli](https://github.com/microsoft/winget-cli/) package manager. This is built-in on Windows 11 and later.
```

Comment:
Super minor, but should this have an article before it? (Seems like either "using winget" or "using the winget package manager" would work, unless "winget package manager" is the actual proper name.)

```suggestion
Install Pulumi using the [winget-cli](https://github.com/microsoft/winget-cli/) package manager. This is built-in on Windows 11 and later.
```